course/src/main/java/org/openedx/course/presentation/section/CourseSectionFragment.kt

```diff
@@ -143,23 +145,40 @@ class CourseSectionFragment : Fragment() {
                         }
                     }
                 )
+
+                LaunchedEffect(rememberSaveable { true }) {
+                    val descendantId = requireArguments().getString(ARG_DESCENDANT_ID, null)
+                    if(descendantId.isNotEmpty()) {
+                        router.navigateToCourseContainer(
+                            requireActivity().supportFragmentManager,
+                            descendantId,
+                            courseId = viewModel.courseId,
+                            courseName = "",
```

Comment:
Is it okay to pass an empty string? 🤔

Reply:
we no longer need `courseName` in descendant views, so I have updated the code accordingly